examples/lib/stories/collision_detection/circles.dart

```diff
@@ -7,11 +7,24 @@ import 'package:flame/geometry.dart';
 import 'package:flame/input.dart';
 import 'package:flutter/material.dart' hide Image, Draggable;
 
-const circlesInfo = '''
-This example will create a circle every time you tap on the screen. It will have
-the initial velocity towards the center of the screen and if it touches another
-circle both of them will change color.
-''';
+class Circles extends FlameGame with HasCollidables, TapDetector {
+  static const description = '''
+    This example will create a circle every time you tap on the screen. It will
+    have the initial velocity towards the center of the screen and if it touches
+    another circle both of them will change color.
+  ''';
+
+  @override
+  Future<void> onLoad() async {
+    super.onLoad();
+    add(ScreenCollidable());
+  }
+
+  @override
+  void onTapDown(TapDownInfo info) {
+    add(MyCollidable(info.eventPosition.game));
+  }
+}
 
 class MyCollidable extends PositionComponent
```

Comment:
Change to `_MyCollidable`, in order to indicate that it's private to this file?

Reply:
Since this isn't library code I don't think this is necessary, and it would also break consistency with most of the other of the examples.